src/app/contextBroker/CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ ELSEIF(${DISTRO} STREQUAL "Fedora_20")
 # specify switch
 
 ELSE()  
-  TARGET_LINK_LIBRARIES(contextBroker ${STATIC_LIBS} -lmongoclient ${BOOST} ${DYNAMIC_LIBS})
+  TARGET_LINK_LIBRARIES(contextBroker ${STATIC_LIBS} ${BOOST} ${DYNAMIC_LIBS})
```

Comment:
Better to keep the default as it was and create and ELSEIF specific for Debian 9

Reply:
Fixed in 08b9789 and beaa423